### PR TITLE
ERMS-4159, ERMS-4157, ERMS-4019

### PR DESCRIPTION
**Pull request by Moetez16**

### CODE REVIEW

File: server/gokbg3/grails-app/domain/org/gokb/cred/IdentifierNamespace.groovy [hunk 1]
```diff
@@ -33,9 +33,8 @@ class IdentifierNamespace {
   }
 
   static constraints = {
-    // II: Want this, but need to tidy live first :: value (nullable:true, blank:false, unique:true)
     name (nullable:true)
-    value (nullable:true, blank:false, unique:true)
+    value (nullable:true, blank:false, unique: false)
```
Comment (croqueGrec09) [inline server/gokbg3/grails-app/domain/org/gokb/cred/IdentifierNamespace.groovy]: make the value (i.e. the key) not unique? Is this not dangerous?

File: server/gokbg3/grails-app/migrations/changelog-2022-03-10.groovy [hunk 2]
```diff
@@ -8,5 +8,9 @@ databaseChangeLog = {
         dropColumn(columnName: "idns_datatype", tableName: "identifier_namespace")
     }
 
+    changeSet(author: "djebeniani (generated)", id: "1646901131691-3") {
```
Comment (croqueGrec09) [inline server/gokbg3/grails-app/migrations/changelog-2022-03-10.groovy:11]: Why?

**Review by croqueGrec09 — COMMENTED** [review]
One remark

**Review by croqueGrec09 — APPROVED** [review]
approved

File: server/gokbg3/grails-app/migrations/changelog-2022-03-10.groovy (hunk 2 above)
Comment (Moetez16) [reply server/gokbg3/grails-app/migrations/changelog-2022-03-10.groovy:11]: idns_datatype

**Pull request merged**